src/abi/rswETH.json

```diff
@@ -0,0 +1,1142 @@
+[
```

Comment:
Might restrict this to minimally expose what we need

Nobody needs to see admin stuff